hostprocess/calico/calico.yml

```diff
@@ -131,7 +131,7 @@ spec:
         # This container installs the CNI binaries
         # and CNI network config file on each node.
         - name: install-cni
-          image: sigwindowstools/calico-install:v3.20.0-hostprocess
+          image: sigwindowstools/calico-install:v3.24.5-hostprocess
```

Comment:
Shouldn't you be using `VERSION` here and elsewhere in this YAML?

Reply:
Forgot to modify those too. I updated them now